SiReCore/utterance_load.py

```diff
@@ -184,44 +185,109 @@ def proto_from_hts_lab(lab):
   proto["id"] = lab[0]
   lab.pop(0)
   # Create split lib
-  delims = ["~","-","+","=",":","_","/A/","_","_","/B/","-","-",":","-","&","-","#","-","$","-",">","-","<","-","|","/C/","+","+","/D/","_","/E/","+",":","+","&","+","#","+","/F/","_","/G/","_","/H/","~",":","=","&","/I/","_","/J/","+","-"]
-  c_word = {"id":"", "syllables":[]}
+
+  delims = ["^","-","+","=","@","_","/A:","_","_","/B:","-","-","@","-","&","-","#","-","$","-","!","-",";","-","|","/C:","+","+","/D:","_","/E:","+","@","+","&","+","#","+","/F:","_","/G:","_","/H:","=","@","=","|","/I:","=","/J:","+","-","[","]"]
```

Comment:
You should make it an optional input argument in the proto_from_hts_lab method to choose between state and phone labels and then act accordingly. For the delimiters please simply use the old ones for "phone" and the new ones for "state"